.github/workflows/deploy.yml

```diff
@@ -46,6 +46,15 @@ jobs:
           tag: 'v${{ steps.compute-new-version.outputs.new_version }}'
           message: 'New Version: ${{ steps.compute-new-version.outputs.new_version  }}'
 
+      - name: Compute requirements-all
```

Comment:
what's this step? 
and why is it called "compute requirements" but it computes the version?!